ConsoleApp6/Products.cs

```diff
@@ -1,14 +1,18 @@
 ﻿using System;
+using System.Collections.Generic;
+
 namespace KATA
 {
-   public class Products 
+    public class Products
     {
         public string Name { get; set; }
         public string UPC { get; set; }
         public double Price { get; set; }
 
         public static int Tax { get; set; }
         public static int Discount { get; set; }
+        public static int UPC_Discount { get; set; }
+        public static List<string> specialUPC = new List<string>() { "1234", "4321" };
 
         public Products(string name,string upc,double price)
```

Comment:
the discounts, tax rates, specialUPC are not part of the Product's real-world representation, not part of it, so we can look for a better way to separate the price calculations data from the product itself.